ruby/build_release.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This file should be executed with ruby
+
+set -ex
+
+WORKING_DIR=$(dirname $(readlink $BASH_SOURCE))
+
+rvm use ruby-3.0
```

Comment:
This requires rvm and ignores the ruby version selected by the caller

Reply:
I thought you told me to use this command? 

What should I do instead here to make sure we are using ruby and not jruby?

Reply:
I said you should use this command from the release script, and specifically not inside a Bazel rule.  Ruby is now bazelified, so Bazel rules should be using whatever version was selected by the user/toolchain resolution. `@rules_ruby//ruby/runtime:interpreter` should resolve to the appropriate binary.

rvm can still be used *outside* Bazel to select a pinned version in our release script.  I was imagining our release script would look something like:
```
rvm use ruby-3.0
bazel build //ruby:release
cp bazel-bin/ruby/<generated release> <dest>
```

Reply:
Note: that assumes our WORKSPACE file selects the system ruby version, similar to our python setup.  After some planned enhancements we would also be able to pass the ruby version as a bazel flag

Reply:
I removed the lines setting ruby/jruby version and am just using `target_compatible_with` to force the correct version.